src/components/SemanticModal/SemanticModal.stories.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+import { action } from '@storybook/addon-actions';
+
+import SemanticModal from './SemanticModal';
+import { SemanticModalVariants } from './SemanticModal.enums';
+import { SemanticModalProps } from './SemanticModal.types';
+import { Inline, Padbox } from '../layout';
+import { Button, ButtonEnums } from '../Button';
+import { SpaceSizes } from '../../theme';
+import { generateControl } from '../../utils/tests/storybook';
+
+export default {
+  title: 'components/modals/SemanticModal',
+  component: SemanticModal,
+  parameters: {
+    docs: {
+      inlineStories: false,
+      iframeHeight: 500,
+      source: { type: 'code' },
+    },
+  },
+  argTypes: {
+    variant: {
+      ...generateControl('select', SemanticModalVariants),
+    },
+    actions: {
+      control: { disable: true },
+    },
+  },
+} as Meta;
+
+export const Playground: Story<SemanticModalProps> = (args) => {
+  const [isShown, setIsShown] = useState(false);
+
+  return (
+    <Padbox paddingSize={SpaceSizes.xxl}>
+      <Inline justify="center">
+        <Button
+          size={ButtonEnums.ButtonSizes.lg}
+          onClick={() => setIsShown(!isShown)}
+        >
+          Open semantic modal
+        </Button>
+      </Inline>
+      {isShown && (
+        <SemanticModal
+          actions={[
+            {
+              label: 'Secondary',
+              name: 'Secondary',
+              onClick: action('click-action-secondary'),
+            },
+            {
+              label: 'Primary',
+              name: 'Primary',
+              onClick: action('click-action-primary'),
+            },
+          ]}
+          message="You can replace this with a local component. You can also just override the text, but keep in mind that you might lose content that way."
+          title="Success"
+          onClose={() => setIsShown(false)}
+          {...args}
+        />
+      )}
+    </Padbox>
+  );
+};
```

Comment:
If you are using `args` you should pass them into the component through the `Playground.args = { ... }` and then you can just use `<SemanticModal {...args} />`  (this is especially valid for the playground story).
You should provide static props only when you want to prevent changing the props by users, but since the published Storybook contains only Docs mode even that is not needed. 
You can take a look at https://github.com/securityscorecard/design-system/blob/alpha/src/components/Dropdown/Dropdown.stories.tsx